pkg/build/apis/build/log_options_test.go

```diff
@@ -4,11 +4,11 @@ import (
 	"reflect"
 	"testing"
 
-	kapi "k8s.io/kubernetes/pkg/apis/core"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
This change doesn't look right.  This package is about internals.

Reply:
yes, switched this back